CHANGES.rst

```diff
@@ -5,6 +5,8 @@ Version 8.0.2
 
 Unreleased
 
+-   Fix shell completion for arguments that start with non-alphanumeric
+    characters such as absolute file paths.
```

Comment:
Keep two newlines between headers, add `:issue:` directive.

Reply:
still missing `:issue:` directive

Reply:
Yeah, working on it...